.docker/entrypoint/lib/commands/jekyll_server.rb

```diff
@@ -15,6 +15,9 @@ module Commands
       class JekyllServer < JekyllBuilder
         def execute
           super
+          puts "<<<------------->>>"
```

Comment:
Codacy found an issue: [Prefer single-quoted strings when you don't need string interpolation or special symbols.](https://app.codacy.com/gh/SwedbankPay/jekyll-plantuml-docker/pullRequest?prid=6123667)